cards-api/README.md

```diff
@@ -23,13 +23,17 @@ Production API Host: `https://api.wise.com`
 
 ### Getting started
 
-The flow has 4 parts: Authentication, Card Issuance, Card Management and Sensitive Card Details.
-
-To generate a card, you need to create a card order. The card order availability API endpoint tells which Card Programs are available for your profile. A Card Program is we call all of the cards you will be issuing with us, grouped by card type and by issuing country. If your profile is verified, the card will be generated in a few seconds. If your profile is not verified, the card order status will be PLACED until your profile gets verified.
-
-The Card Management section lets you retrieve cards (without the PANs), set transactional, daily, monthly and lifetime limits, retrieve the card transactions, temporarily freeze cards and permanently block cards.
-
-The Sensitive Card Details section lets you retrieve the card PANs. Because the request and response are encrypted, we recommend you to use our `get-card-details` Node module instead. Please [contact our support team](mailto:api@transferwise.com) if you don't have the Node module to retrieve card PANs.
+The flow has 10 parts: 
+1. Authentication
+2. Card Issuance: To generate a card, you need to create a card order. The card order availability API endpoint tells which Card Programs are available for your profile. A Card Program is we call all of the cards you will be issuing with us, grouped by card type and by issuing country. If your profile is verified, the card will be generated in a few seconds. If your profile is not verified, the card order status will be PLACED until your profile gets verified. For Preset Card PIN, the request and response are encrypted. We recommend you to use our `preset-pin` Node module instead. Please [contact our support team](mailto:api@transferwise.com) if you don't have the Node module to preset card PINs.
```

Comment:
Maybe good to provide a link to the documentation instead where partner will have more context before contacting us directly? The preset-pin script is a example, they should have enough details in the doc to generate a JWE by themselves

Reply:
Thank you, updated :+1: